app.js

```diff
@@ -61,6 +61,39 @@ class UI {
   }
 }
 
+class Nav {
+  constructor() {
+    this.tabs = document.querySelectorAll('[data-target]');
+  }
+
+  init() {
+    this.tabs.forEach((tab) => {
+      tab.addEventListener('click', (e) => {
+        if (e.target.tagName === 'A') {
+          this.toggleTabs(e);
+          this.toggleContent(e);
+        }
+      });
+    });
+  }
+
+  toggleTabs(e) {
+    // remove current active classes
```

Comment:
- [ ] Please remove comments from your code. For a cleaner code, we encourage avoiding them.